_posts/2023-09-15-kafka-producer-client-essentials.md

```diff
@@ -0,0 +1,323 @@
+---
+layout: post
+title: "Developing Kafka clients: A simple producer client"
+date: 2023-09-15
+author: paul_mellor
+---
+
+Strimzi simplifies the deployment and management of Kafka clusters in a Kubernetes environment, ensuring a smooth and hassle-free experience.
+But the real fun begins once your cluster is up and running.
+That's when you can start thinking about how to interact with Kafka to stream data.
+In other words, Kafka doing the job it excels at.
+
+In this post, we'll dive into the essentials of developing a Kafka producer client that can send messages to a Strimzi-managed Kafka cluster.
```

Comment:
It look like this post is about developing a Kafka producer client _application_, or just Kafka producer application.
For me a "Kafka producer client" is the underlying client that speaks the Kafka protocol, it's an interface between systems. On the other hand, an application achieves something, a use case , a process, for its users. If you agree, then the title should be updated as well and pretty much replace all instances of "client" by "application".

Reply:
Changed to _producer application_.
I think it's better for SEO too, as it appears to be a more commonly used search item.

Reply:
> It look like this post is about developing a Kafka producer client _application_, or just Kafka producer application. For me a "Kafka producer client" is the underlying client that speaks the Kafka protocol, it's an interface between systems. On the other hand, an application achieves something, a use case , a process, for its users. If you agree, then the title should be updated as well and pretty much replace all instances of "client" by "application".

@mimaison @PaulRMellor would this change also make sense in the title? `Developing Kafka clients: A simple producer application` => `Developing Kafka client applications: A simple producer`

Reply:
Good suggestion, I agree

Reply:
@PaulRMellor Do you want to update the title to Jakub's suggestion?